Libero_Projects/import/components/IMC_DGC2/README.md

```diff
@@ -0,0 +1,138 @@
+
+**Note:** This design configuration is only available for the  Polar Fire Avalanche Kit (Revision 3 with production silicon devices). For programming and debugging, this configuration requires the Dual EE Click board to be connected the Avalanche Board mikroBUS headerconnected to the MikroBus header fo
+
+## <a name="quick"></a> Instructions
+
+#### Running Libero SoC in GUI mode, with Script Arguments
+    1. Open Libero SoC
+    2. Execute the selected script, Project -> Execute Script
+    3. Select the directory that the script is located in, using the "..."
+    4. Select the script and select "Open"
+    5. In the arguments text box, enter "DGC2 SYNTHESIZE"
+    6. Select the "Run" button to execute the script
+    7. Once complete, a script report will be generated.
+
+In this example, the arguments "DGC2 SYNTHESIZE" are entered to take the project through to Synthesis.
+
+Libero executes the script and opens the Mi-V sample project. The script adds Timing constraints to the project for Synthesis, Place and Route, and Timing Verification. Additionally, IO Constraints are added to the project for Place and Route. The project can now be taken through the remainder of the Libero SoC design flow.
+
+## <a name="Script arguments"></a> Script Arguments
+The complete set of script arguments are documented here.
+
+#### First argument:
+| Argument                  |  Description   |
+| ------------------------- |:---------------|
+| DGC2                      | Generate a MIV_ESS example design from the MIV_ESS Design Guide  |
+
+#### Second argument:
+| Argument                  |  Description   |
+| ------------------------- |:---------------|
+| SYNTHESIZE                | Run synthesis on the design  |
+| PLACE_AND_ROUTE           | Run place and route on the design  |
+| GENERATE_BITSTREAM        | Generate the bitstream for the design|
+| EXPORT_PROGRAMMING_FILE   | Export the programming file (.job) |
+
+
+### Software Provided
+There are two programs included with this configuration:
+* **miv-rv32i-systick-blinky.hex**: A Hex program configured to run from TCM's address (0x4000_0000). The program is initialized in the LSRAM component at 0x8000_0000 and it is accessible over the AHB interface.
+    > The example hex program was created using  miv-rv32i-systick-blinky in release mode (mivrv32i-Release).
+* **miv-rv32-ess-bootloader.elf**: The supplied elf file is used to copy data from the LSRAM (SRC_MEM) @0x8000_0000 to external flash memory connected to either  I2C or SPI.
+    > The .elf program was compiled using 'miv-rv32-ess-bootloader' in Bootstrap mode.
```

Comment:
miv-rv32-ess-bootloader link to

https://github.com/Mi-V-Soft-RISC-V/miv-rv32-bare-metal-examples/tree/main/applications/bootloaders/miv-rv32-bootloader

Reply:
done